app/models/remember.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+# remmeber
+class Remember < ApplicationRecord
```

Comment:
Change the class name to be more descriptive

Reply:
Fixed